server/src/test/java/org/elasticsearch/search/internal/ContextIndexSearcherTests.java

```diff
@@ -376,6 +379,54 @@ public void onRemoval(ShardId shardId, Accountable accountable) {
         IOUtils.close(reader, w, dir);
     }
 
+    public void testComputeSlices() throws IOException {
+        Directory dir = newDirectory();
+        IndexWriter w = new IndexWriter(dir, newIndexWriterConfig(null));
```

Comment:
Maybe just use `RandomIndexWriter`

Reply:
I changed to use the RIW and simplified the test quite a bit, thanks!